### PR TITLE
Reviving old branch - Ideas/wikinews

### PR DESCRIPTION
**Pull request by iambibhas**
I took one of the branches mentioned in #1535 and worked on it a little. Updated the frontend to latest format and added handlebar template. This is how it looks - 

wikinews_at_duckduckgo_-_2015-03-16_22_17_31

This is the new PR against master branch. The old one #1632 got a bit out of hand because I opened it against the stale `ideas/wikinews` branch.

### CODE REVIEW

File: share/spice/wikinews/content.handlebars [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h5>Latest Articles from Wikinews</h5>
```
Comment (moollaza) [inline share/spice/wikinews/content.handlebars]: Looks like we can remove this file?

**Comment by jagtalon** [comment]
@moollaza looks like @iambibhas made his changes.

**Comment by moollaza** [comment]
@iambibhas I can't remember what we discussed, but just wanted to double check -- is there any chance we can get summaries/snippets or photos for this Spice?

**Comment by iambibhas** [comment]
@moollaza no. :| the api just provides page id, title and timestamp.

**Comment by moollaza** [comment]
@abeyang I think this is ready for a design pass. It's really unfortunate, but this is about all the data we can get from Wiki News...

It just dawned on me that perhaps the list view would be better for this IA?

**Comment by abeyang** [comment]
I think this is a decent foundation, but agree with @moollaza that it would be nice if they can open up their API a bit more (this is bare bones at the moment).

With that said, @iambibhas let's make the tiles thinner and a little bit taller. First, let's modify the css (prefixed with your own namespace, of course):

``` css
.tile--c, .tile--m {
  width: 14em;
  height: 11em;
}
```

And instead of using the class `tile__title--3sm` for the titles, let's use these classes: 
`tile__title--4 tx--17`

In the end, it should look something like this:

screen shot 2015-04-28 at 4 11 08 pm

Thanks!

**Comment by iambibhas** [comment]
@abeyang I added that css change. Also, I'm currently using `tileTitle: "3line-small"` for title. What do I need to do to add that `tile__title--4 tx--17` class instead? Here is how it looks now -

**Comment by moollaza** [comment]
@iambibhas you'll want to use the new `elClass` property to pass along the `tx--17` and `variants` to change the`tileTitle` to `4line`

Details: https://duck.co/duckduckhack/spice_displaying#directly-specifying-classes

**Comment by iambibhas** [comment]
@moollaza doing this is adding the `tx--17` class but not the `tile__title--4` -

```
    templates: {
        group: 'text',
        options: {
            footer: Spice.wikinews.footer
        },
        detail: false,
        item_detail: false,
        variants: {
            tileTitle: "4line",
        },
        elClass: {
            tileTitle: 'tx--17'
        },
    }
```

**Comment by iambibhas** [comment]
The example for `4line` in the [doc](https://duck.co/duckduckhack/spice_templates_overview) is actually using `3line-small` - https://duckduckgo.com/?q=live+shows+in+london&ia=concerts

**Comment by moollaza** [comment]
@iambibhas good catch, sorry about that

@abeyang it looks like we removed the `4line` title internally -- was that expected?

**Comment by moollaza** [comment]
@iambibhas alright, we've re-implemented the `tileTile: 4line` variant -- this should be working again!

**Comment by iambibhas** [comment]
@moollaza it isn't working. :( I ran `duckpan -e -v server` to clear the cache but it's still not appending the `tile__title--4` class.

**Comment by moollaza** [comment]
@iambibhas sorry I spoke to soon, it hasn't been pushed live yet. Should be going on soon though...

**Comment by moollaza** [comment]
LGTM :+1: 

@iambibhas `4line` is now working. I'll add it in with a new PR.

**Pull request merged**